Purpose: Adjust startpar(8) for Debian.
  Correct startpar SEE ALSO section, and Use correct path to runlevel directories.
Author:  Roger Leigh, Petter Reinholdtsen
Fixes:   Missing and unavailable references
Status:  Not submitted upstream
Bug-Debian: http://bugs.debian.org/678260
Forwarded: not-needed
Reviewed-By: Petter Reinholdtsne <pere@hungry.com>
Last-Update: 2014-03-28

Index: startpar-git/startpar.8
===================================================================
--- startpar-git.orig/startpar.8	2014-04-11 23:59:31.317966143 +0200
+++ startpar-git/startpar.8	2014-04-11 23:59:33.453980504 +0200
@@ -109,7 +109,7 @@
 respectively in the directory
 .IR /etc/init.d/ .
 By scanning the boot and runlevel directories in
-.I /etc/init.d/
+.I /etc/rcX.d/
 it then executes the appropriate scripts in parallel.
 
 .SH FILES
@@ -120,9 +120,8 @@
 .I /etc/init.d/.depend.stop
 
 .SH SEE ALSO
-.BR init.d (7),
-.BR insserv (8),
-.BR startproc (8).
+.BR init (8)
+.BR insserv (8).
 
 .SH COPYRIGHT
 2003,2004 SuSE Linux AG, Nuernberg, Germany.
